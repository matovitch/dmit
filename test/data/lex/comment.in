// this is a first comment
// this is a second comment
*/
/*
This
is
a
third
comment
*/
/*
    This is a
    /*
        nested comment
    */
*/

/* toto /*
