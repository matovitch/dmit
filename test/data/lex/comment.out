{"tokens" :["START_OF_INPUT","COMMENT","COMMENT","STAR_SLASH","WHITESPACE","STAR_SLASH","WHITESPACE","STAR_SLASH","WHITESPACE","UNKNOWN","END_OF_INPUT"],"offsets":[157,130,102,100,99,70,69,13,11,0,0]}